content/docs/gke/deploy/deploy-cli.md

```diff
@@ -45,24 +44,27 @@ Follow these steps to deploy Kubeflow:
     export KUBEFLOW_PASSWORD=<your password>
     ```
 
-1. Download a kfctl release from the [GitHub releases](https://github.com/kubeflow/kubeflow/releases/)
-
-1. Unpack the tar ball
+1. Download a `kfctl` release from the 
+  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/).
 
-   ```
-    tar -xvf kfctl_${TAG}_${PLATFORM}.tar.gz
-   ```
+1. Unpack the tar ball:
```

Comment:
@sarahmaddox this might need some trivial updates soon after the release. But for now we can put it as is.